### DISCUSSION TITLE
Show a segment only in home dir

### DISCUSSION THREAD

**Discussion by swp-nil** [JanDeDobbeleer/oh-my-posh#5509] [General]
I am having trouble to find a way to only show a segment in home dir

**Comment by lewis-yeung** [comment]
@KosmoKr3ator You can tweak the template of a segment by checking the current working directory (`.PWD`). For example,

```json
"template": "{{if eq .PWD \"~\"}}YOUR CONTENT HERE{{end}}"
```
